pilot/helpers/Project.py

```diff
@@ -342,8 +360,35 @@ def get_all_coded_files(self):
             )
         )
 
+    def get_all_coded_files(self, relevant_files=None):
+        """
+        Get all coded files in the project.
+
+        Returns:
+            list: A list of coded files.
+        """
+        files = self.get_all_database_files()
+        if relevant_files:
+            n_files0 = len(files)
+            files = [file for file in files if self.relpath(file) in relevant_files]
```

Comment:
This is the actual filtering step. The path normalization (`self.relpath`) is there so we can match file path properly with the relevant files set.